README.md

```diff
@@ -4,21 +4,402 @@ Complete the following queries and paste the sql results under each question. Us
+ORDER BY TotalPrice Desc);
+
+Number of Records: 1
+ID	MAX(TotalPrice)
+10372	15353.6
+
+
 8. Select the names of the employees that have shipped Tofu
+
+We couldn't get this to work with concatenating first and last name. :(
+SELECT Employees.FirstName+' '+Employees.LastName AS Name
+FROM Employees
+JOIN Orders
+ON Employees.EmployeeID=Orders.EmployeeID
+JOIN OrderDetails
+ON Orders.OrderID=OrderDetails.OrderID
+JOIN Products
+ON Products.ProductID=OrderDetails.ProductID
+WHERE Products.ProductName="Tofu"
+
+We got it to work when we selected the first and last columns separately
+SELECT FirstName, LastName
+FROM Employees
+JOIN Orders
+ON Employees.EmployeeID=Orders.EmployeeID
+JOIN OrderDetails
+ON Orders.OrderID=OrderDetails.OrderID
+JOIN Products
+ON Products.ProductID=OrderDetails.ProductID
+WHERE Products.ProductName="Tofu"
+
+Number of Records: 8
+FirstName	LastName
+Michael	Suyama
+Nancy	Davolio
+Steven	Buchanan
+Janet	Leverling
+Nancy	Davolio
+Janet	Leverling
+Laura	Callahan
+Margaret	Peacock
+
 9. Select the name of the customer with the most expensive order
+
+SELECT Name, MAX(TotalPrice)
+FROM (
+SELECT Customers.CustomerName as Name, Orders.OrderID as ID, SUM(OrderDetails.Quantity * Products.Price) as TotalPrice
+FROM Customers
+JOIN Orders
+ON Customers.CustomerID=Orders.CustomerID
+INNER JOIN OrderDetails
+ON Orders.OrderID=OrderDetails.OrderID
+INNER JOIN Products
+ON Products.ProductID = OrderDetails.ProductID
+Group by OrderDetails.OrderID
+ORDER BY TotalPrice Desc);
+
+Number of Records: 1
+Name	MAX(TotalPrice)
+Queen Cozinha	15353.6
+
+
 10. Select all the product names that are in the most expensive order
+
+SELECT * FROM Products
+INNER JOIN OrderDetails
+ON Products.ProductID=OrderDetails.ProductID
+WHERE OrderDetails.OrderID=(SELECT OrderDetails.OrderID
+FROM OrderDetails
+INNER JOIN Products
+ON Products.ProductID = OrderDetails.ProductID
+Group by OrderDetails.OrderID
+ORDER BY SUM(OrderDetails.Quantity * Products.Price) Desc
+LIMIT 1);
+
+Number of Records: 4
+ProductID	ProductName	SupplierID	CategoryID	Unit	Price	OrderDetailID	OrderID	Quantity
+20	Sir Rodney's Marmalade	8	3	30 gift boxes	81	331	10372	12
+38	Côte de Blaye	18	1	12 - 75 cl bottles	263.5	332	10372	40
+60	Camembert Pierrot	28	4	15 - 300 g rounds	34	333	10372	70
+72	Mozzarella di Giovanni	14	4	24 - 200 g pkgs.	34.8	334	10372	42
+
 11. Select the name of the employee with the least amount of orders
+
+SELECT Employees.FirstName, Employees.LastName, COUNT(Orders.OrderID) AS TotalOrders
+FROM Employees
+LEFT JOIN Orders
+ON Employees.EmployeeID=Orders.EmployeeID
+GROUP BY LastName
+ORDER BY TotalOrders;
+
+Number of Records: 10
+FirstName	LastName	TotalOrders
+Adam	West	0
+Anne	Dodsworth	6
+Steven	Buchanan	11
+Robert	King	14
+Michael	Suyama	18
+Andrew	Fuller	20
+Laura	Callahan	27
+Nancy	Davolio	29
+Janet	Leverling	31
+Margaret	Peacock	40
+
 12. Select the number of orders made in each month sorted by the number of orders
+
+//This is not working!! :(
```

Comment:
So this is another one that would require http://www.w3schools.com/sql/func_extract.asp. DatePart is in the right direction too.

There's one more part you're missing in this query, remember COUNT is an aggregate function